### PR TITLE
Add LUTs derived from data taken during DM-30949 run.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: ATAOS/v3/hex_m1_hex_20210706_v3.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Ticket used to create this file: DM-30949
+# Date created: 2021-07-07
+# Description: DM-30949 Support AT observing run
+#   Data taken on the summit still pending validation.
+#   New M1 pressure LUT with data taken on 20210216
+#   New Hexapod LUT TBD
```
Comment (patrickingraham) [inline ATAOS/v3/hex_m1_hex_20210706_v3.yaml]: is this really still TBD?

File: ATAOS/v3/hex_m1_hex_20210706_v3.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Ticket used to create this file: DM-30949
+# Date created: 2021-07-07
```
Comment (patrickingraham) [inline ATAOS/v3/hex_m1_hex_20210706_v3.yaml:2]: this date doesn't match the date in the filename

File: ATAOS/v3/hex_m1_hex_20210707_v3.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Ticket used to create this file: DM-30949
+# Date created: 2021-07-07
+# Description: DM-30949 Support AT observing run
+#   Data taken on the summit still pending validation.
+#   New M1 pressure LUT with data taken on 20210216
+#   New Hexapod LUT TBD
```
Comment (patrickingraham) [inline ATAOS/v3/hex_m1_hex_20210707_v3.yaml]: same comment as above

**Review by patrickingraham — APPROVED** [review]
Address comments before merging please.

File: ATAOS/v3/hex_m1_hex_20210706_v3.yaml (hunk 2 above)
Comment (tribeiro) [reply ATAOS/v3/hex_m1_hex_20210706_v3.yaml:2]: The filename has the date for when the data was taken, this is the date when the file was created... :/

**Pull request merged**